pom.xml

```diff
@@ -364,6 +364,12 @@
                 <version>${apache.commons.version}</version>
                 <scope>compile</scope>
             </dependency>
+            <dependency>
+                <groupId>io.github.nettyplus</groupId>
+                <artifactId>netty-leak-detector-junit-extension</artifactId>
+                <version>0.0.5</version>
```

Comment:
for consistency, can we extract a maven property containing the version.
I think in the case of this library, we can attach a scope of test here too.

Reply:
I'm not a fan of attaching scope in dependency management as its really tricky to resolve downstream (as in to force back to compile in that one awkward corner case) and I think violates the principle of least surprise as a non scoped dependency won't be compile.

I have extracted the property however.